mlflow/server/js/src/common/sdk/UnityCatalogService.js

```diff
@@ -5,4 +5,16 @@ export class UnityCatalogService {
    * List all UC catalogs.
    */
   static listCatalogs = () => getJson({ relativeUrl: 'ajax-api/2.0/unity-catalog/catalogs' });
+
+  /**
```

Comment:
shall we OSS-guard UC-specific code? 🤔

Reply:
Yep :) I'll exclude this file from the sync files.